Source/Parallelization/WarpXComm_K.H

```diff
@@ -68,561 +82,217 @@ void warpx_interp (int j, int k, int l,
     arr_aux(j,k,l) = arr_fine(j,k,l) + res;
 }
 
+/**
+ * \brief Interpolation function called within WarpX::UpdateAuxilaryDataStagToNodal
+ * to interpolate data from the coarse and fine grids to the fine aux grid,
+ * with momentum-conserving field gathering, hence between grids with different staggering.
```

Comment:
Thanks for updating the docstring. Could you add one more sentence, indicating explicitly that the staggering of the `aux` grid is assumed to be nodal.